### PR TITLE
Added floating createThread button for mobile discussions page

### PR DESCRIPTION
**Pull request by kurtisassad**
## Link to Issue
Closes: #5174

## Description of Changes
- Adds create thread button that floats over text in mobile view
- Removes old create thread button when in mobile view

Old (disregard bottom left icon, will not be present on production):
Screenshot 2023-10-17 at 12 53 05 PM

New:
image

image

## Test Plan
- CA

## Other Considerations
- I am not sure the sizing/ spacing for the create thread button

### CODE REVIEW

**Comment by kurtisassad** [comment]
NOTE: Do not accept until we get design/product sign off

**Review by dillchen — CHANGES_REQUESTED** [review]
Add to Overview page as well

**Comment by kurtisassad** [comment]
> Add to Overview page as well
Added

**Comment by dillchen** [comment]
per Kurtis' notes there, we want this also on the community homepage but that's after Community Homepage gets merged in

**Review by dillchen — APPROVED** [review]

File: packages/commonwealth/client/scripts/controllers/app/login.ts
Comment (Miaplacidus) [inline packages/commonwealth/client/scripts/controllers/app/login.ts:400]: Is this line required given that an empty object is the default return for the function?

**Comment by Miaplacidus** [comment]
@kurtisassad
Great stuff! Left a couple comments. Also, there's an unneeded import in `HeaderWithFilters`.

**Review by Miaplacidus — CHANGES_REQUESTED** [review]

**Comment by dillchen** [comment]
- right: 12px;
- bottom: 12px;
- level three shadow

**Review by Miaplacidus — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx
Comment (kurtisassad) [reply packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx:1]: Yeah removed. We want to add creation of a thread on the user_dasboard in the future, but not in this PR.

**Comment by mhagel** [comment]
has conflicts and approvals

**Pull request merged**
